devtools/x-lint/src/runner.rs

```diff
@@ -206,67 +197,6 @@ impl<'cfg> LintEngine<'cfg> {
 
         Ok(LintResults { skipped, messages })
     }
-
-    // ---
-    // Helper methods
-    // ---
-
-    fn get_file_list(&self) -> Result<Vec<&Path>> {
-        // TODO: It'd be pretty cool to be able to cache the `&Path` instances as well. The options
-        // are:
-        // 1. Cache them as `PathBuf` instances. That would involve lots of allocations.
-        // 2. Store the `&Path` instances alongside `files_stdout`. That would be a self-referential
-        //    struct (so would require the use of something like
-        //    [`rental`](https://docs.rs/rental/).
-        // 3. Reorganize the code to use some sort of arena allocator which is passed into the lint
-        //    context (and/or stored alongside with `rental`.)
-        let stdout = self.files_stdout()?;
-
-        Ok(stdout
-            .split(|b| b == &0)
-            .filter_map(|s| {
-                // TODO: make global exclusions configurable.
-                if !s.is_empty() && !s.starts_with(b"testsuite/libra-fuzzer/artifacts/") {
```

Comment:
I think you still need this filter, otherwise any automation/attempt to add fuzz locations will fail on commit.

Reply:
Oh yeah, thanks for catching this. Will fix.

Reply:
Done.